demo/AngryBots/index.html

```diff
@@ -1,124 +1,13 @@
-  var screenHeight = inHiDPIFullscreenMode ? Math.round(window.innerHeight*window.devicePixelRatio) : window.innerHeight;
-//  var screenWidth = screen.width * window.devicePixelRatio;
-//  var screenHeight = screen.height * window.devicePixelRatio;
-  var w = screenWidth;
-  var h = screenHeight;
-  var canvas = Module['canvas'];
-  var x = canvas.width;
-  var y = canvas.height;
-  var topMargin;
-
-  if (inAspectRatioFixedFullscreenMode) {
-    if (w*y < x*h) h = (w * y / x) | 0;
-    else if (w*y > x*h) w = (h * x / y) | 0;
-    topMargin = ((screenHeight - h) / 2) | 0;
-  }
-
-  if (inPixelPerfectFullscreenMode) {
-    canvas.width = w;
-    canvas.height = h;
-    console.log('new canvas size: ' + w + 'x' + h);
-    if (canvas.GLctxObject) canvas.GLctxObject.GLctx.viewport(0, 0, canvas.width, canvas.height);
-  }
-
-  // Back to CSS pixels.
-  if (inHiDPIFullscreenMode) {
-    topMargin /= window.devicePixelRatio;
-    w /= window.devicePixelRatio;
-    h /= window.devicePixelRatio;
-    // Round to nearest 4 digits of precision.
-    w = Math.round(w*1e4)/1e4;
-    h = Math.round(h*1e4)/1e4;
-    topMargin = Math.round(topMargin*1e4)/1e4;
-  }
-/*
-  if (inCenteredWithoutScalingFullscreenMode) {
-    var t = (window.innerHeight - parseInt(canvas.style.height)) / 2;
-    var b = (window.innerWidth - parseInt(canvas.style.width)) / 2;
-    __setLetterbox(canvas, t, b);
-  } else {
-    canvas.style.width = w + 'px';
-    canvas.style.height = h + 'px';
-    var b = (window.innerWidth - w) / 2;
-    __setLetterbox(canvas, topMargin, b);
-  }
-*/  
-  canvas.style.width = w + 'px';
-  canvas.style.height = h + 'px';
-}
-
-document.getElementById('canvas').style.width = window.innerWidth + 'px';
-document.getElementById('canvas').style.height = window.innerHeight + 'px';
-document.getElementById('canvas').width = window.innerWidth * window.devicePixelRatio;
-document.getElementById('canvas').height = window.innerHeight * window.devicePixelRatio;
-window.addEventListener('resize', softFullscreenResizeWebGLRenderTarget);
-</script>
-
-<script type='text/javascript'>
-  // Note: all this feature testing is only necessary to serve old versions of
-  // the binary format and JS API to old experimental browser builds. None of
-  // this would be necessary in a production WebAssembly build.
-  var version = "";
-  if (typeof Wasm === 'object' && Wasm.experimentalVersion < 0xc) {
-    version = String(Wasm.experimentalVersion);
-  } else if (typeof WebAssembly === 'object') {
-    if (WebAssembly.validate(Uint8Array.of(0x0, 0x61, 0x73, 0x6d, 0x01, 0x00, 0x00, 0x00))) {
-        version = String(0x1);
-    } else if (WebAssembly.validate(Uint8Array.of(0x0, 0x61, 0x73, 0x6d, 0x0c, 0x00, 0x00, 0x00))) {
-        version = String(0xc);
-    } else if (WebAssembly.validate(Uint8Array.of(0x0, 0x61, 0x73, 0x6d, 0x0d, 0x00, 0x00, 0x00))) {
-        version = String(0xd);
-    }
-  }
```

Comment:
Are we losing the feature-tested older versions? I'd at least note this in the PR  description.